src/dag/topological-sort.js

```diff
@@ -112,30 +112,34 @@ function forEachTopologicalGeneration(graph, callback) {
   let lastGenLevel = 0;
   let lastGen = [];
 
-  forEachNodeInTopologicalOrder(graph, (node, _, gen) => {
+  function loopBody(node, _, gen) {
     if (gen > lastGenLevel) {
       callback(lastGen);
       lastGenLevel = gen;
       lastGen = [];
     }
 
     lastGen.push(node);
-  });
+  }
+
+  forEachNodeInTopologicalOrder(graph, loopBody, root);
```

Comment:
Moved the loop body out of this call here and below so that readers wouldn't miss the third argument at the end.